io/zenoh-links/zenoh-link-tls/src/utils.rs

```diff
@@ -269,6 +274,11 @@ impl TlsClientConfig {
             root_cert_store.extend(custom_root_cert.roots);
         }
 
+        // Install rustls provider
+        rustls::crypto::ring::default_provider()
+            .install_default()
+            .ok();
```

Comment:
I think it's better to return an error if this operation fails.
```suggestion
            map_err(|err| zerror!("Error installing TLS crypto provider: {err}."))?;
```